packages/webapp/pages/squads/new.tsx

```diff
@@ -79,22 +71,9 @@ const NewSquad = (): ReactElement => {
     <ManageSquadPageContainer>
       <NextSeo {...seo} titleTemplate="%s | daily.dev" noindex nofollow />
       <ManageSquadPageMain>
-        <div
-          style={{
-            backgroundImage: `url(${cloudinary.squads.createSquad})`,
-          }}
-          className="mb-6 flex h-52 w-full flex-col items-center justify-center bg-cover"
-        >
-          <SourceIcon size={IconSize.XXXLarge} />
-          <SquadTitle className="mb-2">Create new Squad</SquadTitle>
-          <SquadSubTitle>
-            Create a group where you can learn and interact privately with other
-            developers around topics that matter to you
-          </SquadSubTitle>
-        </div>
         <SquadDetails
           className="p-8 pt-0"
-          form={form}
+          form={{ public: false }}
```

Comment:
What does this setting do?

Reply:
Not really sure what happens inside, but I just noticed a local state above and doesn't seem to be changed. Instead I added this inline object over the unnecessary hook.